h/api/models/document.py

```diff
@@ -37,7 +37,7 @@ def title(self):
         titles = [m.value for m in self.meta if m.type == 'title']
 
         if titles:
-            return titles[0]
+            return titles[0][0]
```

Comment:
And, I would rather stay on the safe side and not risk an `IndexError` here.

The only reason this method exists is to have a consistent API with the `elastic.Document` if I remember correctly. And I [recently](https://github.com/hypothesis/h/commit/6ea11a67f4a12f399eb82ddf5e3c72670a64cfa0) changed this very method in `elastic.Document` but forgot to change the PG one.

It should behave the same way, so maybe it should look something along the lines of:
```python
@property
def title(self):
    titles = [m.value for m in self.meta if m.type == 'title']
    try:
        return titles[0][0]
    except IndexError:
        return None
```